hbase_sql/src/main/java/com/splicemachine/stream/RemoteQueryClientImpl.java

```diff
@@ -174,16 +174,16 @@ private void updateLimitOffset() throws StandardException {
     }
 
     @Override
-    public void waitForCompletion(int time, TimeUnit unit) throws InterruptedException, TimeoutException, ExecutionException {
+    public void interrupt() {
         streamListener.stopAllStreams();
         if (olapFuture != null)
-            olapFuture.get(time, unit);
+            olapFuture.cancel(true);
     }
 
     @Override
     public void close() throws Exception {
         streamListener.stopAllStreams();
         if (olapFuture != null)
-            olapFuture.cancel(true);
+            olapFuture.cancel(false);
```

Comment:
Interrupt() and close() seem to be the same, as the parameter true or false for the cancel() function is not used. Is it intended to keep two separate functions so that in the future different operations will be applied?

Reply:
You are right, I thought we were handling it in OlapFuture.cancel() and my limited testing didn't reproduce the exception from SPLICE-1895  

I addressed it now by ignoring the Cancel message when we cancel without interrupt.

Reply:
Thanks. Now that remoteQueryClient.close() is used for successful execution, interrupt() is used for timeout or killed scenario, should we change the call of remoteQueryClient.close() in SpliceBaseOperation.kill() and timeout() to interrupt() instead?

Reply:
Yep, good catch. Thanks for the review!